### PR TITLE
Onboard Common ui operator

### PR DESCRIPTION
**Pull request by ericabr**
Adding commonui operator to catalog and config CRs

### CODE REVIEW

**Comment by horis233** [comment]
https://quay.io/application/opencloudio/ibm-commonui-operator-app?tab=settings
ibm-commonui-operator-app is still private, please set it to public.

**Comment by ericabr** [comment]
@horis233 Done

File: deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml [hunk 1]
```diff
@@ -39,3 +39,11 @@ spec:
     sourceName: opencloud-operators
     sourceNamespace: openshift-marketplace
     description: Operator for managing deployment of health check service.
+  - name: ibm-commonui-operator
+    namespace: ibm-commonui-operator
```
Comment (horis233) [inline deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml]: Please change the namespace to `ibm-common-services`. All the common services will be deployed in the same namespace

**Comment by horis233** [comment]
Please follow this PR https://github.com/IBM/meta-operator/pull/86/files to add your changes into the `alm-example` field of the CSV.

**Comment by ericabr** [comment]
@horis233 done

File: deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -12,6 +12,15 @@ metadata:
           },
           "spec": {
             "operators": [
+              {
+                "channel": "alpha",
+                "description": "The service used to display common header and IAM pages for the cluster",
+                "name": "ibm-commonui-operator",
+                "namespace": "ibm-commonui-operator",
```
Comment (horis233) [inline deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml]: ibm-common-services

Comment (ericabr) [reply deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml]: updated!

File: deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml [hunk 3]
```diff
@@ -20,3 +20,7 @@ spec:
   - name: ibm-healthcheck-operator
     spec:
       healthService: {}
+  - name: ibm-commonui-operator
+      spec:
```
Comment (horis233) [inline deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml]:
```
  - name: ibm-commonui-operator
    spec:
      commonWebUI: {}
      legacyHeader: {}
```
There should be empty space before `spec:`

**Comment by horis233** [comment]
@ericabr I can't install your operator from openshift Operatorhub, I think you lost your crd when you push your Operator bundle

**Comment by aniruddhj** [comment]
@horis233 The commonui-operator installs from the openshift OperatorHub. Can you please try again and let us know if there is any issue, thanks!

**Comment by DanielXLee** [comment]
@ericabr Now this operator is `absent` state by default, so this onboard looks good to me, I will approve it at first, and then you can e2e test with OperatorHub

**Pull request merged**